ui/src/components/styles.ts

```diff
@@ -57,14 +57,19 @@ export const ImportButtonWrapper = styled.div`
   }
 `;
 
+const disableLastHeaderCellStylingForSpotlight = css`
+  th:last-child {
+    padding: 0;
+  }
+`;
+
 export const TableWrapper = styled.div`
   margin-top: ${gridSize() * 4}px;
   max-height: 60vh;
   overflow: auto;
-  &::-webkit-scrollbar {
-    width: ${token('space.025', '2px')};
-    border: ${token('space.025', '2px')} solid ${token('color.border.input', '#091E4224')};
-  }
+  padding-right: 12px;
+
+  ${disableLastHeaderCellStylingForSpotlight}
```

Comment:
What does the spotlight look like if we don't remove the styling of the last header?